# point in spherical coordinates, in the camera frame
uint16 angle_vertical       # up-down angle
uint16 angle_horizontal     # left-right angle
uint16 distance_log         # log of distance
uint8 point_level           # the level of the octree where this point came from

# color
uint8 r
uint8 g
uint8 b
